pytorch_lightning/utilities/imports.py

```diff
@@ -88,3 +88,7 @@ def _compare_version(package: str, op, version) -> bool:
 _TORCHTEXT_AVAILABLE = _module_available("torchtext")
 _TORCHVISION_AVAILABLE = _module_available('torchvision')
 _XLA_AVAILABLE = _module_available("torch_xla")
+
+from pytorch_lightning.utilities.xla_device import XLADeviceUtils  # noqa: E402
```

Comment:
Move the import to the top? Also note that the `_*_AVAILABLE` vars are sorted

Reply:
We couldn't do that. As `XLADeviceUtils` imports `_XLA_AVAILABLE`.